src/rustdoc/membership.md

```diff
@@ -0,0 +1,119 @@
+  Conduct][CoC].
+
+[rust-lang.org web page]: https://www.rust-lang.org/governance/teams/dev-tools#team-rustdoc
+
+## What it means to be a rustdoc member
+Once you're a member of the rustdoc team, a number of events will happen:
+
+- You will gain access to a private Zulip stream, where internal discussions happen.
+- You will also be subscribed to the `all@rust-lang.org` and `rustdoc@rust-lang.org` mailing lists.
+  See [this file](https://github.com/rust-lang/team/blob/HEAD/teams/all.toml) to check how
+  subscriptions to mailing lists work. Both are very low-volume mailing list (maybe a few emails per
+  year). About `all@rust-lang.org`: it's a way to communicate things to all contributors. We will
+  not send you spam from this address.
+
+## Roles in the rustdoc team
+
+Rustdoc has multiple different areas and team members are not working on all of them. Currently
+we have three main areas:
+
+- Front-end: Everything related to HTML/CSS/JS
+- JSON backend: Work on the `--output-format=json` backend.
+- Internals: The internals of rustdoc: interacting with the compiler, doctests, generating
+  rustdoc internal code representation, parsing command line arguments, lints, etc.
+
+These groups are NOT full-fledged teams, and as such, to be part of any of these groups, you need to
+be a member of the rustdoc team.
+
+For now, only the front-end group has an official status in the
+[team repository](https://github.com/rust-lang/team) and is called `rustdoc-frontend`. If a rustdoc
+team member is interested to be part of this group, they can ask to be added into it.
+
+Let's take the front-end group as an example. It is a part of the rustdoc team, so you need to be a
+member of the rustdoc team to be able to join this group. Being part of the front-end group means
+you are encouraged to be part of the review rotations for front-end pull requests and you will need
+to respond on front-end FCPs.
+
+## How promotion decisions are made
+[hdam]: #how-promotion-decisions-are-made
+
+After an individual has been contributing to rustdoc for a while, they may be nominated in the
+private Zulip rustdoc team channel by an existing team member. All nominations **must** be done in
+the private Zulip rustdoc team channel.
+
+The rustdoc team members will check to see if there are concerns with extending a membership
+invitation to the individual and after 10 days (barring no objections), an invitation will be
+extended.
+
+If the invitation is accepted by the individual, the rustdoc team leads will update the [team]
+repository to reflect their new role.
+
+## Alumni status
+If at any time a rustdoc team member wishes to take a break from participating, they can opt to put
+themselves into alumni status. When in alumni status, they will be removed from
+GitHub aliases and the like, so that they need not be bothered with pings and messages. They will
+also not have r+ privileges. **Alumni members will however still remain members of the GitHub
+org overall.**
```

Comment:
Side note: IIUC the last sentence will no longer be true after https://github.com/rust-lang/team/pull/1868.

However, let's keep this as we can always amend it once that linked PR actually gets merged.